hw/drivers/bq27z561/include/bq27z561/bq27z561.h

```diff
@@ -232,6 +294,60 @@ int bq27z561_get_temp(struct bq27z561 *dev, float *temp_c);
  */
 int bq27z561_get_voltage(struct bq27z561 *dev, uint16_t *voltage);
 
+/**
+ * bq27z561 set low voltage set threshold
+ *
+ * Sets low voltage value that triggers interrupt
+ *
+ * @param dev pointer to device
+ * @param voltage voltage, in mV
+ *
+ * @return int 0: success, -1 error
+ */
+int bq27z561_set_voltage_lo_set_threshold(struct bq27z561 *dev,
+        uint16_t voltage);
+
+/**
+ * bq27z561 set low voltage clear threshold
+ *
+ * When interrupt is active and voltage goes above this threshold interrupt
+ * is cleared.
+ *
+ * @param dev pointer to device
+ * @param voltage voltage, in mV
+ *
+ * @return int 0: success, -1 error
+ */
+int bq27z561_set_voltage_lo_set_threshold(struct bq27z561 *dev,
+        uint16_t voltage);
+
+/**
+ * bq27z561 set high voltage set threshold
+ *
+ * Sets high voltage value that triggers interrupt
+ *
+ * @param dev pointer to device
+ * @param voltage voltage, in mV
+ *
+ * @return int 0: success, -1 error
+ */
+int bq27z561_set_voltage_hi_set_threshold(struct bq27z561 *dev,
+        uint16_t voltage);
+
+/**
+ * bq27z561 set high voltage clear threshold
+ *
+ * When interrupt is active and voltage goes below this threshold interrupt
+ * is cleared.
+ *
+ * @param dev pointer to device
+ * @param voltage voltage, in mV
+ *
+ * @return int 0: success, -1 error
+ */
+int bq27z561_set_voltage_hi_set_threshold(struct bq27z561 *dev,
+        uint16_t voltage);
```

Comment:
Same comment as above. This is the same prototype as the one above. Is it mis-named?

Reply:
fixed